geomesa-core/src/main/scala/org/locationtech/geomesa/core/util/GeoMesaBatchWriterConfig.scala

```diff
@@ -0,0 +1,57 @@
+package org.locationtech.geomesa.core.util
+
+import java.util.concurrent.TimeUnit
+
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.accumulo.core.client.BatchWriterConfig
+
+import scala.util.Try
+
+object GeoMesaBatchWriterConfig extends Logging {
+  val WRITERLATENCYSECONDS = "geomesa.batchwriter.latency.seconds"  // Measured in seconds
```

Comment:
I think the convention in scala is to use UpperCamelCase for constants and in java its all upper with underscores...so perhaps ```WRITER_LATENCY_SECONDS```